osu.Game/Screens/Play/HUD/HitErrorMeters/HitErrorMeter.cs

```diff
@@ -67,6 +73,8 @@ protected Color4 GetColourForHitResult(HitResult result)
             }
         }
 
+        public abstract void Clear();
```

Comment:
needs xmldoc (explaining that this gets called on seek and what implementors should do about it)

Reply:
This only needs to be `protected`.

Reply:
This will break tests (unless there's something I'm missing with how my tests are written).

Reply:
Fine to leave it public then, I guess.